Title: 349509 – dev-vcs/hgsubversion-1.2.1 fails tests with Python 2.7.1*
URL: https://bugs.gentoo.org/show_bug.cgi?id=349509

Go to: 
		Gentoo Home
Documentation
Forums
Lists
Bugs
Planet
Store
Wiki
Get Gentoo!





Gentoo's Bugzilla – Bug 349509
dev-vcs/hgsubversion-1.2.1 fails tests with Python 2.7.1*
Last modified: 2011-04-21 23:03:35 UTC node [vulture]


Home
| New–[Ex]
| Browse
| Search
| Privacy Policy

| 





[?]
| Reports

| 
Requests

| 
Help


| 
New Account


| 
Log In





[x]



| 
Forgot Password

Login:




[x]













Bug 349509 
      - dev-vcs/hgsubversion-1.2.1 fails tests with Python 2.7.1*


Summary:
dev-vcs/hgsubversion-1.2.1 fails tests with Python 2.7.1*
    








Status:
    

RESOLVED
          WORKSFORME
      






Alias:


        None
    





Product:

Gentoo Linux




Classification:

Unclassified




Component:

Current packages

  (show other bugs)



Hardware:

All
        Linux
      







Importance:
      
High
       enhancement
      


Assignee:

Dirkjan Ochtman (RETIRED)








URL:







Whiteboard:




Keywords:








Depends on:







Blocks:









 





      Reported:
    
2010-12-23 17:03 UTC by Arfrever Frehtes Taifersar Arahesis (RETIRED)





      Modified:
    
2011-04-21 23:03 UTC
      (History)
    




          CC List:
        

1 
          user
          
            (show)
          



python









See Also:






Package list:







Runtime testing required:

---























      Attachments
    




hgsubversion-1.2:20101223-165609.log

              (hgsubversion-1.2:20101223-165609.log,7.40 KB,
                text/plain)

            
2010-12-23 17:04 UTC,

            Arfrever Frehtes Taifersar Arahesis (RETIRED)




Details





View All

Add an attachment
        (proposed patch, testcase, etc.)
    








Note
              You need to
              log in
              before you can comment on or make changes to this bug.
            

















Description


Arfrever Frehtes Taifersar Arahesis (RETIRED)






          2010-12-23 17:03:53 UTC
        

dev-vcs/hgsubversion-1.2 fails tests with Python 2.7.




Comment 1


Arfrever Frehtes Taifersar Arahesis (RETIRED)






          2010-12-23 17:04:36 UTC
        

Created attachment 257875 [details]
hgsubversion-1.2:20101223-165609.log




Comment 2


Dirkjan Ochtman (RETIRED)






          2010-12-23 18:16:47 UTC
        

Is this with 2.7 or 2.7.1? One of the hgsubversion devs says it passes for him, with 2.7.1.




Comment 3


Arfrever Frehtes Taifersar Arahesis (RETIRED)






          2010-12-23 22:32:10 UTC
        

(In reply to comment #2)

2.7.2_pre20101219




Comment 4


Arfrever Frehtes Taifersar Arahesis (RETIRED)






          2011-04-13 17:59:51 UTC
        

I no longer can reproduce this bug with Python 2.7.2_pre20110410. I was still able to reproduce this bug with Python 2.7.2_pre20110227.




Comment 5


Arfrever Frehtes Taifersar Arahesis (RETIRED)






          2011-04-21 23:03:35 UTC
        

Strangely I cannot reproduce this bug even after downgrading Python 2.7.









Format For Printing
 - XML
 - Clone This Bug
 - Clone In The Same 
                        Product
 - Top of page 







Home
| New–[Ex]
| Browse
| Search
| Privacy Policy

| 





[?]
| Reports

| 
Requests

| 
Help


| 
New Account


| 
Log In





[x]



| 
Forgot Password

Login:




[x]

